content/07.pathogenesis.md

```diff
@@ -73,8 +73,7 @@ B) The physical structure of the coronavirus virion, including the components de
 
 #### Pathogenic Mechanisms of Coronaviruses
 
-While, like most viruses, it is possible that SARS-CoV-1 and SARS-CoV-2 can enter cells through endocytosis, coronaviruses are also able to target cells for entry through fusion with the plasma membrane [@doi:10.1038/cr.2008.15; @doi:10.1146/annurev-biochem-060208-104626].
-This process is conserved among coronaviruses and is closely associated with the content of their genomes.
+While, like most viruses, it is possible that SARS-CoV-1 and SARS-CoV-2 can enter cells through endocytosis, coronaviruses are also able to target cells for entry through fusion with the plasma membrane [@doi:10.1038/cr.2008.15; @doi:10.1146/annurev-biochem-060208-104626], a process conserved among coronaviruses.
```

Comment:
```suggestion
While, like most viruses, it is possible that SARS-CoV-1 and SARS-CoV-2 can enter cells through endocytosis,  a process conserved among coronaviruses enables them to target cells for entry through fusion with the plasma membrane [@doi:10.1038/cr.2008.15; @doi:10.1146/annurev-biochem-060208-104626].
```
I was thinking this might be what the reviewer meant, but yours could also be! I just figured I'd put it here to compare our thoughts!

Reply:
As I read back their comment, they say previous sentence, so I think you are correct.